app/src/main/java/com/alexbar/layoutbasic/WelcomeScreen.kt

```diff
@@ -0,0 +1,193 @@
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+
+// Constraint Layout
+import androidx.constraintlayout.compose.ChainStyle
+import androidx.constraintlayout.compose.ConstraintLayout
+import androidx.constraintlayout.compose.Dimension
+
+// Project-Specific Imports
+import com.alexbar.layoutbasic.ui.theme.Dimens
+import com.alexbar.layoutbasic.ui.theme.Typography
+
+@Composable
+fun WelcomeScreen() {
+
+    ConstraintLayout(modifier = Modifier.fillMaxSize()) {
+        val (backgroundBox, optionsIcon, gymIcon, gymText, welcomeText,
+            signInButton, signUpButton, buttonsContainer,
+            socialMediaText, instagramIcon, twitterIcon, facebookIcon
+        ) = createRefs()
+
+        Box(modifier = Modifier
+            .fillMaxSize()
+            .background(createHorizontalGradient())
+            .constrainAs(backgroundBox) {}
+        )
+
+        Icon(
+            imageVector = Icons.Default.MoreHoriz,
+            contentDescription = null,
+            tint = Color.White,
+            modifier = Modifier
+                .padding(end = Dimens.sign_up_padding_32)
+                .size(Dimens.sign_up_icon_more_size)
+                .constrainAs(optionsIcon) {
+                    top.linkTo(parent.top, Dimens.sign_up_constraint_40)
+                    end.linkTo(parent.end)
+                }
+        )
+
+        val topGuideline = createGuidelineFromTop(Dimens.sign_up_top_guideline)
+
+        Icon(
+            imageVector = Icons.Default.SportsMartialArts,
+            contentDescription = null,
+            tint = Color.White,
+            modifier = Modifier
+                .size(Dimens.sign_up_icon_gym_size)
+                .constrainAs(gymIcon) {
+                    top.linkTo(topGuideline)
+                    end.linkTo(parent.end)
+                    start.linkTo(parent.start)
+                }
+        )
+
+        Text(
+            text = "FITNESS CLUB",
+            style = Typography.titleLarge,
+            modifier = Modifier
+                .constrainAs(gymText) {
+                    top.linkTo(gymIcon.bottom, Dimens.sign_up_constraint_24)
+                    end.linkTo(parent.end)
+                    start.linkTo(parent.start)
+                }
+        )
+
+        ConstraintLayout(modifier = Modifier.constrainAs(buttonsContainer) {
+            top.linkTo(gymText.bottom, Dimens.sign_up_constraint_70)
+            bottom.linkTo(socialMediaText.top, Dimens.sign_up_constraint_70)
+            height = Dimension.fillToConstraints
+        }) {
+            Text(
+                text = "Welcome Back",
+                style = Typography.titleLarge,
+                modifier = Modifier
+                    .constrainAs(welcomeText) {
+                        end.linkTo(parent.end)
+                        start.linkTo(parent.start)
+                    }
+            )
+
+            Button(
```

Comment:
for having Background color for Button you can use this code :
            val mainButtonColor = ButtonDefaults.buttonColors(
                containerColor = Color.Red,
                contentColor = Color.Black
            )
            
            and call it into button :
                        Button(
                colors = mainButtonColor,
                modifier = Modifier
                    .fillMaxWidth()
                    .height(Dimens.sign_up_button_height_55)
                    .padding(horizontal = Dimens.sign_up_padding_32)
                    .constrainAs(signUpButton) {
                    },
                onClick = { }
            ) {
                Text(
                    text = "SIGN IN",
                    style = Typography.titleSmall
                )
            }
            createVerticalChain(welcomeText, signInButton, signUpButton, chainStyle = ChainStyle.Spread)

        }